code/modules/projectiles/projectile/bullets.dm

```diff
@@ -294,6 +294,47 @@
 	embed = 0
 	sharp = 0
 
+/obj/item/projectile/bullet/pistol/papadov
+	name = "bullet"
+	damage = 20
+	agony = 0
+	weaken = 0
+
+
+/obj/item/projectile/bullet/pistol/papadov/tranq
+	name = "tranquilizer"
+	damage = 1
+	damage_type = BRUTE
+	speed = 0.3
+
+/obj/item/projectile/bullet/pistol/papadov/tranq/on_hit(var/atom/target, var/blocked = 0, var/def_zone = null)
+	var/mob/living/L = target
+	if(!(isanimal(target)))
+		if(!(isipc(target)))

```

Comment:
Collapse some of this stuff. `if (!isipc(target) && !isrobot(target))` for example.